src/components/tl/ucp/tl_ucp.c

```diff
@@ -36,6 +36,12 @@ ucc_config_field_t ucc_tl_ucp_lib_config_table[] = {
     {"", "", NULL, ucc_offsetof(ucc_tl_ucp_lib_config_t, super),
      UCC_CONFIG_TYPE_TABLE(ucc_tl_lib_config_table)},
 
+    {"ALLGATHER_LINEAR_NUM_POSTS", "auto",
```

Comment:
is this paramater read in the non-batch version ?

Reply:
no, only batched version need that, will rename for clarity